pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/participants/Osteo1ParticipantCreator.java

```diff
@@ -0,0 +1,557 @@
+
+        QuestionAnswersUtil.answerTextQuestion("CONSENT_ASSENT_CHILD_LASTNAME", participantGuid, activityDto.getGuid(),
+                lastName, answerDao);
+
+        List<SelectedPicklistOption> optionsRelation = new ArrayList<>();
+        optionsRelation.add(new SelectedPicklistOption("PARENT"));
+        QuestionAnswersUtil.answerPickListQuestion("CONSENT_ASSENT_RELATIONSHIP", participantGuid, activityDto.getGuid(),
+                optionsRelation, answerDao);
+
+        QuestionAnswersUtil.answerTextQuestion("CONSENT_ASSENT_CHILD_SIGNATURE", participantGuid, activityDto.getGuid(),
+                "sign consent", answerDao);
+
+        jdbiActivityInstanceStatus.insert(activityDto.getId(), InstanceStatusType.COMPLETE, ddpCreatedAt + 1000,
+                activityDto.getParticipantId());
+    }
+
+    String getMedicalProviderGuid(Handle handle) {
+        return DBUtils.uniqueStandardGuid(handle,
+                SqlConstants.MedicalProviderTable.TABLE_NAME, SqlConstants.MedicalProviderTable.MEDICAL_PROVIDER_GUID);
+    }
+
+    private void populateRelease(Handle handle, String participantGuid, AnswerDao answerDao,
+                                 JdbiActivityInstanceStatus jdbiActivityInstanceStatus, ActivityInstanceDto activityDto,
+                                 long userId, long studyId, String agreementStableId) throws Exception {
+
+        QuestionAnswersUtil.answerAgreementQuestion(agreementStableId, participantGuid, activityDto.getGuid(),
+                true, answerDao);
+
+        String guid = getMedicalProviderGuid(handle);
+        MedicalProviderDao medicalProviderDao = handle.attach(MedicalProviderDao.class);
+        medicalProviderDao.insert(new MedicalProviderDto(
+                null,
+                guid,
+                userId,
+                studyId,
+                InstitutionType.INSTITUTION,
+                "Brigham And Women's Hospital",
+                "Dr Pepper Jr",
+                "Boston",
+                "MA",
+                "UNITED STATES",
+                null,
+                null,
+                null,
+                null
+        ));
+
+        guid = getMedicalProviderGuid(handle);
+        medicalProviderDao.insert(new MedicalProviderDto(
+                null,
+                guid,
+                userId,
+                studyId,
+                InstitutionType.PHYSICIAN,
+                "Massachusetts General Hospital",
+                "Dr Pepper",
+                "Boston",
+                "MA",
+                "UNITED STATES",
+                null,
+                null,
+                null,
+                null
+        ));
+
+        jdbiActivityInstanceStatus.insert(activityDto.getId(), InstanceStatusType.COMPLETE, ddpCreatedAt + 2000,
+                activityDto.getParticipantId());
+    }
+
+    private UserRegistrationResponse registerGovernedUser(Handle handle, GovernedUserRegistrationPayload payload, LocalDate birthDate,
```

Comment:
Thought of moving this to utils too but didnt want to risk someone invoking it witout any Authorization check in here